pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Display a list of hosts belonging to a group:
+
+`ansible {{group}} --list-hosts`
+
+- Ping a group of hosts:
+
+`ansible {{group}} -m ping`
+
+- Display facts about a group of hosts:
+
+`ansible {{group}} -m setup`
+
+- Execute a command (using the command module) on a group of hosts:
+
+`ansible {{group}} -m command -a '{{my command}}'`
```

Comment:
I updated the placeholder command to `{{my command}}` to:

- Help users visually differentiate between the argument (`{{command}}`) and the module (`-m command`)
- Hint that it's okay to have spaces in the argument

We could go one step further and use a concrete example (e.g. `{{echo hello world}}`) to unambiguously demonstrate what's allowed, though I'll need a second opinion on that.